gateway-developer-plugin/src/main/java/com/ca/apim/gateway/cagatewayconfig/BuildFullBundleTask.java

```diff
@@ -86,9 +86,9 @@ public void perform() {
         metaDataFiles.stream().forEach(metaDataFile-> {
             final Pair<String, Map<String, String>> bundleEnvironmentValues = environmentConfigurationUtils.parseBundleMetadata(metaDataFile, configFolder.getAsFile().get());
             if (null != bundleEnvironmentValues) {
-                final String bundleFileName = bundleEnvironmentValues.getLeft() + "." + configName.get() + ".full.bundle";
+                final String bundleFileName = bundleEnvironmentValues.getLeft() + "-full" + INSTALL_BUNDLE_EXTENSION;
                 final List<File> bundleFiles = union(
-                        collectFiles(bundleDirectory, bundleEnvironmentValues.getLeft() + BUNDLE_EXTENSION),
+                        collectFiles(bundleDirectory, INSTALL_BUNDLE_EXTENSION),
```

Comment:
If multiple entities are annotated, this may collect all the policy bundles, full annotated bundle should contain only annotated policy bundle and corresponding env entities, just check

Reply:
This code was not being used. Removed it.